KsApi/MockService.swift

```diff
@@ -170,12 +170,15 @@
 
     fileprivate let unwatchProjectMutationResult: Result<
       GraphMutationWatchProjectResponseEnvelope,
-      GraphError
+      ErrorEnvelope
```

Comment:
Although the migration to Apollo, for this method, is not included here, I had to update the function signature because the view model was consolidating some of the logic between `watch` and `unwatch`.  This was needed for compilation.